apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -22,26 +38,74 @@ if (!('BridgeServiceMixin' in self)) {
   ]);
 
   var ConversationService = {
+    mobileMessageClient: null,
+
     /**
      * Initializes our service using the Service mixin's initService.
      */
     init() {
       this.initService();
+
+      this.mobileMessageClient = bridge.client(
+        'mozMobileMessageShim',
+        new BroadcastChannel('mobile-message-shim-channel')
+      ).plugin(streamClient);
     },
 
     /**
      * Stream that returns everything needed to display conversations in the
      * inbox view.
-     * @param {ServiceStream.<ConversationSummary>} The stream to use in the
-     * implementation: it will be passed autoamtically by the Bridge library.
+     * @param {ServiceStream.<ConversationSummary>} serviceStream The stream to
+     * use in the implementation: it will be passed automatically by the Bridge
+     * library.
      */
-    getAllConversations(stream) {},
+    getAllConversations(serviceStream) {
+      var getThreadsStream = this.mobileMessageClient.stream('getThreads');
```

Comment:
> However I think we should call Drafts.request() as the first thing, because this will start the IDB request under the hood.

Althought we also need the thread request to start as early as possible... grr I don't know then :D